.gitmodules

```diff
@@ -22,3 +22,9 @@
 [submodule "projects/1000GenomesProject"]
   path = projects/1000GenomesProject
   url = https://github.com/conpdatasets/1000GenomesProject.git
```

Comment:
Hi @emmetaobrien shouldn't you update this url instead of adding a new one? Is this repo (`1000GenomesProject`) no longer good?

Reply:
I was getting weird failures on trying to update it, hence the new project instead.